### PR TITLE
fix: UI improvements

### PR DESCRIPTION
**Pull request by pwltr**
### Description

- fix(suggestions): add pulsing glow animation
- fix(lightning): keep screen awake when setting up connection

For the animated glow I had trouble to get skia `opacity` to work properly so I'm wrapping with `Animated` and use animation there.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactoring (improving code without creating new functionality)

### Tests

- [ ] Detox test
- [ ] Unit test
- [x] No test

### Screenshot / Video

https://github.com/synonymdev/bitkit/assets/8538369/53483b49-195e-48a1-ad9f-54eda25c4397

### CODE REVIEW

**Review by limpbrains — APPROVED** [review]
This is so cool!

File: src/screens/Lightning/SettingUp.tsx [hunk 1]
```diff
@@ -25,19 +32,25 @@ const SettingUp = ({
 		{ title: t('setting_up_step4') },
 	];
 
-	useEffect(() => {
-		let interval = setTimeout(() => {
-			navigation.navigate('Timeout');
-		}, 2 * 60 * 1000);
+	useFocusEffect(
+		useCallback(() => {
+			activateKeepAwake();
```
Comment (limpbrains) [inline src/screens/Lightning/SettingUp.tsx:37]: So it only works on SettingUp screen, but I can close it

Comment (pwltr) [reply src/screens/Lightning/SettingUp.tsx:37]: You are suggesting we add this to the whole app if channel is setting up?

Comment (limpbrains) [reply src/screens/Lightning/SettingUp.tsx:37]: Yes, this would be better

Comment (pwltr) [reply src/screens/Lightning/SettingUp.tsx:37]: Will look into this and add another PR.

**Pull request merged**